rholang/src/main/scala/coop/rchain/rholang/build/CompiledRholangSource.scala

```diff
@@ -22,7 +22,7 @@ object CompiledRholangSource {
          #$fileContent
          #""".stripMargin('#')
     }
-    override val term: Par = ParBuilder[Coeval].buildNormalizedTerm(code).value()
+    override val term: Par = ParBuilder[Coeval].buildNormalizedTerm(code, None).value()
```

Comment:
I've got to say, the results of having the deployerPk a non-default parameter are far better than expected :astonished:. It's *so painfully clear* where we might be relying on it, and yet don't have it. Superb. I also learnt a thing or two about our codebase :joy: